Service/ServiceProxy.cs

```diff
@@ -47,12 +47,14 @@ private void AddRequestHeaders(HttpRequestMessage message, ServiceOptions option
             }
         }
 
-        private Uri BuildUrl(string path, string queryString, InitServiceOptions options)
+        private Uri BuildUrl(string path, string queryString, ServiceOptions options)
         {
+            var prefixedPath = string.IsNullOrEmpty(options?.Prefix) ? path : $"{options.Prefix}/{path}";
+            
             var uriBuilder = new UriBuilder
             {
                 Host = ServiceName,
-                Path = path,
+                Path = string.IsNullOrEmpty(InitOptions.Prefix) ? prefixedPath : $"{InitOptions.Prefix}/{prefixedPath}",
```

Comment:
I see the reason of the now missing `/`. Is there a library for path joining that is safer than this?

Reply:
I added a path joining library